Failure running `pnpm run test:single ...` in `langchain/community`

### Checked other resources

- [x] This is a bug, not a usage question. For questions, please use the LangChain Forum (https://forum.langchain.com/).
- [x] I added a very descriptive title to this issue.
- [x] I searched the LangChain.js documentation with the integrated search.
- [x] I used the GitHub search to find a similar question and didn't find it.
- [x] I am sure that this is a bug in LangChain.js rather than my code.
- [x] The bug is not resolved by updating to the latest stable version of LangChain (or the specific integration package).

### Example Code

```bash
langchain-community % pnpm run test:single ibm
```

### Error Message and Stack Trace (if applicable)

> @langchain/community@1.0.1 test:single /Users/filipzmijewski/Desktop/nodejs/langchainjs/libs/langchain-community
> NODE_OPTIONS=--experimental-vm-modules pnpm run jest --config jest.config.cjs --testTimeout 100000 ibm

 ERR_PNPM_NO_SCRIPT  Missing script: jest

### Description

Unable to run `test:single` since it contains `ppm run jest` inside.

I proposed change here in PR: https://github.com/langchain-ai/langchainjs/pull/9277

### System Info
```
pnpm info langchain 

langchain@1.0.2 | MIT | deps: 5 | versions: 342
Typescript bindings for langchain
https://github.com/langchain-ai/langchainjs/tree/main/langchain/

keywords: llm, ai, gpt3, chain, prompt, prompt engineering, chatgpt, machine learning, ml, openai, embeddings, vectorstores

dist
.tarball: https://registry.npmjs.org/langchain/-/langchain-1.0.2.tgz
.shasum: c58ec42ed2c9b8ec3d3f3ed02a033a5be885ec58
.integrity: sha512-He/xvjVl8DHESvdaW6Dpyba72OaLCAfS2CyOm1aWrlJ4C38dKXyTIxphtld8hiii6MWX7qMSmu2EaUwWBx2STg==
.unpackedSize: 2.3 MB

dependencies:
@langchain/langgraph-checkpoint: ^1.0.0 langsmith: ~0.3.74                      zod: ^3.25.76 || ^4                     
@langchain/langgraph: ^1.0.0            uuid: ^10.0.0                           

maintainers:
- christian-bromann <mail@bromann.dev>
- nfcampos <nuno@boringbits.io>
- jacoblee93 <jacoblee93@gmail.com>
- andrewnguonly <andrewnguonly@gmail.com>
- davidduong <david@duong.cz>
- hntrl <hunter@hntrl.io>
- hwchase17 <hw.chase.17@gmail.com>
- basproul <braceasproul@gmail.com>

dist-tags:
alpha: 1.0.0-alpha.9                       next: 1.0.0-alpha.9                        
latest: 1.0.2                              tag-for-publishing-older-releases: 0.2.20  

published 2 days ago by hntrl <hunter@hntrl.io>
```
```
node -v
v20.19.2
```
```
pnpm --version
10.18.3
```

--- a/libs/langchain-community/package.json
+++ b/libs/langchain-community/package.json
@@ -22,7 +22,7 @@
     "clean": "rm -rf .turbo dist/",
     "test": "NODE_OPTIONS=--experimental-vm-modules jest --testPathIgnorePatterns=\\.int\\.test.ts --testPathIgnorePatterns=arcjet --testPathIgnorePatterns=srt --testPathIgnorePatterns=standard --testTimeout 30000 --runInBand",
     "test:watch": "NODE_OPTIONS=--experimental-vm-modules jest --watch --testPathIgnorePatterns=\\.int\\.test.ts",
-    "test:single": "NODE_OPTIONS=--experimental-vm-modules pnpm run jest --config jest.config.cjs --testTimeout 100000",
+    "test:single": "NODE_OPTIONS=--experimental-vm-modules jest --config jest.config.cjs --testTimeout 100000",
     "test:int": "NODE_OPTIONS=--experimental-vm-modules jest --testPathPattern=\\.int\\.test.ts --testTimeout 100000 --runInBand",
     "test:standard:unit": "NODE_OPTIONS=--experimental-vm-modules jest --testPathPattern=\\.standard\\.test.ts --testTimeout 100000 --runInBand",
     "test:standard:int": "NODE_OPTIONS=--experimental-vm-modules jest --testPathPattern=\\.standard\\.int\\.test.ts --testTimeout 100000 --runInBand",

